Dockerfile

```diff
@@ -13,28 +13,31 @@ RUN go get github.com/tools/godep
 RUN echo "Installing the go-bindata tool"
 RUN go get github.com/jteeuwen/go-bindata/...
 
-ADD . /src/github.com/
+ADD . /src/github.com/openwhisk/openwhisk-cli
+
+#RUN git clone http://github.com/openwhisk/openwhisk-cli /src/github.com/openwhisk/openwhisk-cli
+
+#RUN cd /src/github.com/openwhisk/openwhisk-cli && git checkout master
 
 # Load all of the dependencies from the previously generated/saved godep generated godeps.json file
 RUN echo "Restoring Go dependencies"
-RUN cd /src/github.com/go-whisk-cli && /bin/godep restore -v
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/godep restore -v
 
 # Collect all translated strings into single .go module
 RUN echo "Generating i18n Go module"
-RUN cd /src/github.com/go-whisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
-RUN cd /src/github.com/go-whisk && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
+RUN cd /src/github.com/openwhisk/openwhisk-cli && /bin/go-bindata -pkg wski18n -o wski18n/i18n_resources.go wski18n/resources
 
 # Generate a Go package dependency list
 # NOTE: Currently, the 'go list' command will not work against the current Go CLI non-standard file structure
 #RUN cd /src/github.com/go-whisk-cli && go list -f '{{join .Deps "\n"}}' > ../../../wsk.deps.out
-RUN cd /src/github.com/go-whisk-cli && echo "Dependencies list requires restructuring the GO CLI packages" > ../../../wsk.deps.out
+RUN cd /src/github.com/openwhisk/openwhisk-cli && echo "Dependencies list requires restructuring the GO CLI packages" > ../../../wsk.deps.out
```

Comment:
I think if we would like to have the real wsk.deps.out, we need to comment out Line 32 and remove Line 33, since we have the go project structure. We are able to run go list command.